model/generate.go

```diff
@@ -275,17 +276,57 @@ func (g *GeneratedProject) saveNewBuildsAndTasks(ctx context.Context, v *Version
 	return nil
 }
 
-func (g *GeneratedProject) findVariantsToActivate() []string {
-	var toActivate []string
+type batchTimeTasksAndVariants struct {
+	tasks    map[string][]string // tasks by variant that have batchtime
+	variants []string            // variants that have batchtime
+}
+
+func newBatchTimeTasksAndVariants() batchTimeTasksAndVariants {
+	return batchTimeTasksAndVariants{
+		tasks:    map[string][]string{},
+		variants: []string{},
+	}
+}
+
+func (b *batchTimeTasksAndVariants) isBatchTimeVariant(variant string) bool {
```

Comment:
I think I made this same comment on the previous pr so sorry for being a broken record, but can you rename this function so it's a bit clearer? Maybe something like variantHasBatchtime?

Reply:
haha sure